roles/openshift_logging/tasks/upgrade_logging.yaml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Stop the Cluster
+  include: stop_cluster.yaml
+
+- name: Upgrade logging
+  include: install_logging.yaml
+  vars:
+    start_cluster: False
+
+# ensure that ES is running
+- shell: >
+    {{openshift.common.client_binary}} --config={{mktemp.stdout}}/admin.kubeconfig get dc -l component=es -o name -n {{openshift_logging_namespace}}
+  register: es_dc
+  check_mode: no
+
+- name: start elasticsearch
+  include: scale.yaml
+  vars:
+    desired: 1
```

Comment:
Actually, this is as intended. Given that there aren't petsets yet, we want to start up all ES instances so that we can run a migrate script against them. So we would need each ES DC to at least be at 1 replica. Later on when we start the cluster, we should be able to handle scaling up the other components to their correctly specified replica count where applicable